src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -3509,6 +3525,7 @@ private void onApplicationDataContOrFin(
                     OctetsFW payload)
                 {
                     final boolean more = (flags & FLAG_FIN) == 0;
+                    final boolean aborted = (flags & FLAG_INIT) == FLAG_INIT;
```

Comment:
This should be `FLAG_INCOMPLETE` instead of `FLAG_INIT`, agreed?
`onApplicationDataInit` should be able to handle a `DATA` frame with both `FLAG_INIT` and `FLAG_INCOMPLETE` set, but there's no reference to `aborted` in there at the moment.

Reply:
Yes, sorry I meant to have `FLAG_INCOMPLETE` there.